packages/nx/src/project-graph/utils/project-configuration-utils.ts

```diff
@@ -189,13 +189,14 @@ export type ConfigurationResult = {
  * Transforms a list of project paths into a map of project configurations.
  *
  * @param nxJson The NxJson configuration
- * @param projectFiles A list of files identified as projects
+ * @param workspaceFiles A list of non-ignored workspace files
  * @param plugins The plugins that should be used to infer project configuration
  * @param root The workspace root
  */
 export function buildProjectsConfigurationsFromProjectPathsAndPlugins(
   nxJson: NxJsonConfiguration,
-  projectFiles: string[], // making this parameter allows devkit to pick up newly created projects
+  // TODO: clarify this comment
+  workspaceFiles: string[], // making this parameter allows devkit to pick up newly created projects
```

Comment:
Can we rename this `allConfigurationFiles` or similar? This isn't all workspace files, and there's already a different meaning for `workspaceFiles` which would be all of them.

This property contains all files that have been identified as a configuration file, so we should name it to line up with that